[RHACS] Add external database requirements documentation (#93890)
openshift/openshift-docs: modules/default-requirements-external-db.adoc

// Module included in the following assemblies:
//
// * installing/acs-default-requirements.adoc
:_mod-docs-content-type: CONCEPT
[id="external-db-req_{context}"]
= Requirements for using an external database

[role="_abstract"]
You can configure {product-title} Central services to use an external PostgreSQL-compatible database for data persistence instead of deploying its own database pod.

[IMPORTANT]
====

> When you use an external database:

Reviewer: ```suggestion
When you use an external database, note the following guidance:
```

> * Its infrastructure manages persistent storage for Central DB. Therefore, do not configure persistence settings for the Central DB within the {product-title-short} installation configuration.

Reviewer: ```suggestion
* The database infrastructure manages persistent storage for Central DB. Therefore, do not configure persistence settings for the Central DB within the {product-title-short} installation configuration.
```

> * Red Hat supports the configuration and operation of {product-title-short} Central connected to that database. However, support for database-specific operations such as backup and restore, performance diagnosis and potential tuning, software and version upgrades, and HA/DR operation falls under 3rd Party Support. Manually upgrading or customizing your database outside of a full platform upgrade also limit supportability.

Reviewer: ```suggestion
* Red Hat supports the configuration and operation of {product-title-short} Central connected to that database. However, support for database-specific operations such as backup and restore, performance diagnosis and potential tuning, software and version upgrades, and high availability/disaster recovery operation falls under third-party support. Manually upgrading or customizing your database outside of a full platform upgrade also limits supportability.
```

====

> If you select an external database, your database instance and the user connecting to it must meet the following requirements:

Reviewer: Since these are headings that follow, I think that this should be a complete sentence. Maybe:

```suggestion
If you select an external database, your database instance and the user connecting to it must meet the requirements listed in the following sections.
```

[discrete]
== Database type and version

> The database must be a PostgreSQL-compatible database that supports PostgreSQL 13 or newer.

Reviewer: ```suggestion
The database must be a PostgreSQL-compatible database that supports PostgreSQL 13 or later.
```
